### DISCUSSION TITLE
CPU - 10 of 12 Maxed

### DISCUSSION THREAD

**Discussion by 520kaymann** [Stellarium/stellarium#3069] [General]
Using Stellarium Version 1.22.5 Completely love the program.

Issue: CPU Core 10 of 12 is full throttle (98-100%) 100% of the time. Overall CPU usage is 14%

Question: 1) does Stellarium support multicore load spreading? 2) does Stellarium support GPU offloading?

If either is yes how do I enable it?

Again FANTASTIC program.

Randall

**Comment by gzotti** [comment]
(Moved to #3070)
